examples/helmet_detection/README.md

```diff
@@ -0,0 +1,192 @@
+      parameters:
+        - key: "batch_size"
+          value: "32"
+        - key: "epochs"
+          value: "1"
+        - key: "input_shape"
+          value: "352,640"
+        - key: "class_names"
+          value: "person,helmet,helmet-on,helmet-off"
+        - key: "nms_threshold"
+          value: "0.4"
+        - key: "obj_threshold"
+          value: "0.3"
+    trigger:
+      checkPeriodSeconds: 60
+      timer:
+        start: 02:00
+        end: 04:00
+      condition:
+        operator: ">"
+        threshold: 500
+        metric: num_of_samples
+  evalSpec:
+    workerSpec:
+      scriptDir: "/code"
+      scriptBootFile: "eval.py"
+      frameworkType: "tensorflow"
+      frameworkVersion: "1.15"
+      parameters:
+        - key: "input_shape"
+          value: "352,640"
+        - key: "class_names"
+          value: "person,helmet,helmet-on,helmet-off"
+  deploySpec:
+    model:
+      name: "deploy-model"
+    trigger:
+      condition:
+        operator: ">"
+        threshold: 0.1
+        metric: precision_delta
+    nodeName: "cloud0"
+    hardExampleMining:
+      name: "IBT"
+    workerSpec:
+      scriptDir: "/code"
+      scriptBootFile: "inference.py"
+      frameworkType: "tensorflow"
+      frameworkVersion: "1.15"
+      parameters:
+        - key: "input_shape"
+          value: "352,640"
+        - key: "video_url"
+          value: "rtsp://localhost/video"
+        - key: "HE_SAVED_URL" 
+          value: "/he_saved_url"
+  nodeName: "cloud0"
+  outputDir: "/output"
+EOF
+```
+1. The `Dataset` describes data with labels and `HE_SAVED_URL` indicates the address of the deploy container for uploading hard examples. Users will mark label for the hard examples in the address.
+2. Ensure that the path of outputDir in the YAML file exists on your node. This path will be directly mounted to the container.
+
+
+
+### Mock Video Stream for Inference in Edge Side
+
+* step1: install the open source video streaming server [EasyDarwin](https://github.com/EasyDarwin/EasyDarwin/tree/dev).
+* step2: start EasyDarwin server.
+* step3: download [video](https://edgeai-neptune.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/video.tar.gz).
+* step4: push a video stream to the url (e.g., `rtsp://localhost/video`) that the inference service can connect.
+
+```
+wget https://github.com/EasyDarwin/EasyDarwin/releases/download/v8.1.0/EasyDarwin-linux-8.1.0-1901141151.tar.gz --no-check-certificate
+tar -zxvf EasyDarwin-linux-8.1.0-1901141151.tar.gz
+cd EasyDarwin-linux-8.1.0-1901141151
+./start.sh
+
+mkdir -p /data/video
+cd /data/video
+tar -zxvf video.tar.gz
+ffmpeg -re -i /data/video/helmet-detection.mp4 -vcodec libx264 -f rtsp rtsp://localhost/video
+```
+
+
+### Check Incremental Job Result
+
+query the service status
+```
+kubectl get incrementallearningjob helmet-detection-demo -n neptune-test
+```
+
+after the job completed, we can view the updated model in the /output directory in cloud0 node
+
```

Comment:
Pls add the section about how/when to incremental learning

Reply:
had add the section